pyccel/ast/core.py

```diff
@@ -201,78 +199,6 @@ def subs(expr, new_elements):
     else:
         return expr
 
-
-def allocatable_like(expr, verbose=False):
```

Comment:
Could you add a line about the removal of this function in the PR description please